build.gradle

```diff
@@ -161,45 +163,51 @@ allprojects {
     testAnnotationProcessor(betterStringsCoords)
   }
 
-  gradle.projectsEvaluated {
```

Comment:
This `gradle.projectsEvaluated` didn't seem to have any effect here, probably my mindless copy-paste from stackoverflow